docs/model_library/operational_water_management/index.rst

```diff
@@ -593,42 +589,76 @@ Flow is only allowed in a given direction if the binary indicator for that direc
 
     \textcolor{red}{y_{l,\tilde{l},t}^{Flow}}+\textcolor{red}{y_{\tilde{l},l,t}^{Flow}} = 1
 
-.. note:: Technically this constraint should only be enforced for truly reversible arcs (e.g. NCA and CNA); and even then it only needs to be defined per one reversible arc (e.g. NCA only and not NCA and CNA).
-
 .. math::
 
-    \textcolor{red}{F_{l,l,t}^{Piped}}\leq \textcolor{red}{y_{l,l,t}^{Flow}} \cdot \textcolor{green}{M^{Flow}}
+    \textcolor{red}{F_{l,\tilde{l},t}^{Piped}} \leq \textcolor{red}{y_{l,\tilde{l},t}^{Flow}} \cdot \textcolor{green}{M^{Flow}}
 
 
 
 **Storage Site Balance:** :math:`\forall s \in S, t \in T`
 
 For each storage site and for each time period, if it is the first time period, the storage level is the initial storage. Otherwise, the storage level is equal to the storage level in the previous time period plus water inputs minus water outputs.
 
-.. math::
+For :math:`t = 1`:
 
-    \textcolor{red}{L_{s,t}^{Storage}} = \textcolor{green}{\lambda_{s,t=1}^{Storage}}+\textcolor{red}{L_{s,t-1}^{Storage}}+\sum\nolimits_{(n,s)\in NSA}\textcolor{red}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(p,s)\in PST}\textcolor{red}{F_{l,l,t}^{Trucked}}
+.. math::
 
-        +\sum\nolimits_{(p,s)\in CST}\textcolor{red}{F_{l,l,t}^{Trucked}}-\sum\nolimits_{(s,n)\in SNA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,p)\in SCA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,k)\in SKA}\textcolor{red}{F_{l,l,t}^{Piped}}
+    \textcolor{red}{L_{s,t}^{Storage}}
+        = \textcolor{green}{\lambda_{s,t=1}^{Storage}}
+        + \sum_{l \in L | (l, s) \in LLA}\textcolor{red}{F_{l,s,t}^{Piped}}
+        + \sum_{l \in L | (l, s) \in LLT}\textcolor{red}{F_{l,s,t}^{Trucked}}
+        - \sum_{l \in L | (s, l) \in LLA}\textcolor{red}{F_{s,l,t}^{Piped}}
+        - \sum_{l \in L | (s, l) \in LLT}\textcolor{red}{F_{s,l,t}^{Trucked}}
+        - \textcolor{red}{F_{s,t}^{StorageEvaporationStream}}
 
-        -\sum\nolimits_{(s,r)\in SRA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,o)\in SOA}\textcolor{red}{F_{l,l,t}^{Piped}}-\sum\nolimits_{(s,p)\in SCT}\textcolor{red}{F_{l,l,t}^{Trucked}}-\sum\nolimits_{(s,k)\in SKT}\textcolor{red}{F_{l,l,t}^{Trucked}}
+For :math:`t > 1`:
 
+.. math::
 
+    \textcolor{red}{L_{s,t}^{Storage}}
+        = \textcolor{red}{L_{s,t-1}^{Storage}}
+        + \sum_{l \in L | (l, s) \in LLA}\textcolor{red}{F_{l,s,t}^{Piped}}
+        + \sum_{l \in L | (l, s) \in LLT}\textcolor{red}{F_{l,s,t}^{Trucked}}
+        - \sum_{l \in L | (s, l) \in LLA}\textcolor{red}{F_{s,l,t}^{Piped}}
+        - \sum_{l \in L | (s, l) \in LLT}\textcolor{red}{F_{s,l,t}^{Trucked}}
+        - \textcolor{red}{F_{s,t}^{StorageEvaporationStream}}
```

Comment:
F^{StorageEvaporationStream} does not appear in the code, so I think it should be removed from the documentation.

Reply:
Addressed.